commando/application.py

```diff
@@ -305,10 +305,10 @@ def run(self, args=None):
                 args.run(self, args)
             else:
                 self.__main__(args) # pylint: disable-msg=E1101
-        except Exception, e: # pylint: disable-msg=W0703
+        except Exception as e: # pylint: disable-msg=W0703
             import traceback
             self.logger.debug(traceback.format_exc())
-            self.logger.error(e.message)
+            self.logger.error(str(e))
```

Comment:
`e.message` doesn't exist in Python 3. `str(e)` behaves correctly under both languages.